lib/catalog/owid/catalog/tables.py

```diff
@@ -530,3 +578,638 @@ def join(self, other: Union[pd.DataFrame, "Table"], *args, **kwargs) -> "Table":
                 t._fields[k] = dataclasses.replace(v)
                 t._fields[k].sources = [dataclasses.replace(s) for s in v.sources]
         return t  # type: ignore
+
+    def merge(self, right, *args, **kwargs) -> "Table":
+        return merge(left=self, right=right, *args, **kwargs)
+
+    def melt(
+        self,
+        id_vars: Optional[Union[Tuple[str], List[str], str]] = None,
+        value_vars: Optional[Union[Tuple[str], List[str], str]] = None,
+        var_name: str = "variable",
+        value_name: str = "value",
+        *args,
+        **kwargs,
+    ) -> "Table":
+        return melt(
+            frame=self,
+            id_vars=id_vars,
+            value_vars=value_vars,
+            var_name=var_name,
+            value_name=value_name,
+            *args,
+            **kwargs,
+        )
+
+    def pivot(
+        self,
+        *,
+        index: Optional[Union[str, List[str]]] = None,
+        columns: Optional[Union[str, List[str]]] = None,
+        values: Optional[Union[str, List[str]]] = None,
+        join_column_levels_with: Optional[str] = None,
+        **kwargs,
+    ) -> "Table":
+        return pivot(
+            data=self,
+            index=index,
+            columns=columns,
+            values=values,
+            join_column_levels_with=join_column_levels_with,
+            **kwargs,
+        )
+
+    def underscore(self) -> "Table":
+        from .utils import underscore_table
+
+        return underscore_table(self, inplace=False)
```

Comment:
`underscore_table` has other arguments (`collision`, `camel_to_snake`), which I think should be accessible from this class method. Perhaps with a `kwargs`.